### DISCUSSION TITLE
Add custom domain name with DigitalOcean

### DISCUSSION THREAD

**Discussion by hugoschreib** [pocketbase/pocketbase#2313] [Q&A] [answered]
I'm running a pocketbase server on a DigitalOcean droplet with the pocket base image. When I run `systemctl start pocketbase`, the server is accessible from "ip_adress":80/_/ . 
However, I have a custom domain name corretly setup on DigitalOcean and I want to acces pockebase from this domain name,
the documentation says : To add a custom domain, update the reverse proxy here: `/etc/caddy/Caddyfile`. 
What do I need to put in this file ?
I tried : `domain_name:80` but it doesn't work

**Comment by ganigeorgiev** [comment, marked as answer]
I assume you mean the [DigitalOcean 1-Click app](https://marketplace.digitalocean.com/apps/pocketbase).
Please note that this is not an officially maintained image and I cannot provide support for it.

Based on the 1-Click app description and its [packer script](https://github.com/scott/pocketbase-do-packer), the Caddy file could be found at `/etc/caddy/Caddyfile`.
You can check https://caddyserver.com/docs/quick-starts/https or explore further the [Caddy reverse proxy documentation](https://caddyserver.com/docs/quick-starts/reverse-proxy) on the possible configurable options.

If you are not using the 1-Click app, please make sure to check https://pocketbase.io/docs/going-to-production/#minimal-setup.
